src/main/kotlin/no/nav/syfo/dialogmote/tilgang/DialogmoteTilgangService.kt

```diff
@@ -32,7 +33,7 @@ class DialogmoteTilgangService(
             token = token,
             callId = callId,
         ).filter { personIdentNumber ->
-            !adressebeskyttelseClient.hasAdressebeskyttelse(personIdentNumber, callId)
+            kode6Enabled || !adressebeskyttelseClient.hasAdressebeskyttelse(personIdentNumber, callId)
```

Comment:
Det er jo få folk, så det har kanskje ikke så mye å si, men hadde det vært mer effektivt å gjøre `kode6Enabled || filter`?

Reply:
Er egentlig den ekstra filtreringen nødvendig? Skal ikke `veilederTilgangskontrollClient.hasAccessToPersonList` returnere en komplett liste med identer som veilederen har tilgang til, inkludert kode6? 🤔

Reply:
Fikset litt på det nå. Men ideen er at så lenge det er togglet av så skal ting virke som før (dvs ikke tilgang til kode 6 i det hele tatt).

Reply:
Ja, det kan kanskje fjernes helt når vi ikke lenger trenger toggle 👍 Det bare ser ut for meg som at det aldri har vært nødvendig 🤷

Reply:
Ja, enig i at det har vært dobbelt opp slik det har vært.